docs/docusaurus/docs/core/introduction/community_resources.md

```diff
@@ -1,8 +1,22 @@
 ---
-title: Contribute
-id: contributing
+title: 'Community Resources'
 ---
+import GxCloudAdvert from '/static/docs/_static_components/_gx_cloud_advert.md'
 
+Great Expectations (GX) is committed to supporting and growing the GX community. It’s not enough to build a great tool. GX wants to build a great community as well.
+
+Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and GX is dedicated to making you feel welcome.
+
+## Contact Great Expectations
+Join the GX [public Slack channel](https://greatexpectations.io/slack). Before you post for the first time, review the [Slack Guidelines](https://discourse.greatexpectations.io/t/slack-guidelines/1195).
+
+## Ask a question
+Slack is good for that, too: [join Slack](https://greatexpectations.io/slack) and read [How to write a good question in Slack](https://github.com/great-expectations/great_expectations/discussions/4951). You can also use [GitHub Discussions](https://github.com/great-expectations/great_expectations/discussions/4951).
+
+## File a bug report or feature request
+If you have bugfix or feature request, see [upvote an existing issue](https://github.com/great-expectations/great_expectations/issues) or [open a new issue](https://github.com/great-expectations/great_expectations/issues/new).
+
+## Contribute code or documentation
 To contribute to Great Expectations documentation or code, see one of the following resources:
 
 - To request a documentation change, or a change that doesn't require local testing, see the [README](https://github.com/great-expectations/great_expectations/tree/develop/docs) in the `docs` repository.
```

Comment:
I'm not seeing a way to comment on some of the lines below, so in reference to these lines (26-28):
```
To create and submit a Custom Expectation to Great Expectations for consideration, see [CONTRIBUTING_EXPECTATIONS](https://github.com/great-expectations/great_expectations/blob/develop/CONTRIBUTING_EXPECTATIONS.md) in the great_expectations repository.

To submit a custom package to Great Expectations for consideration, see [CONTRIBUTING_PACKAGES](https://github.com/great-expectations/great_expectations/blob/develop/CONTRIBUTING_PACKAGES.md) in the great_expectations repository.
```

Are we still planning on accepting Custom Expectations or packages right away when 1.0 launches?

Reply:
Good question.  I don't believe we are; I'll comment out those lines for now and check with Timber.